sunpy/net/fido_factory.py

```diff
@@ -158,6 +158,24 @@ def response_block_properties(self):
             s.intersection(self.get_response(i).response_block_properties())
         return s
 
+    @property
+    def table(self):
+        """
+        Returns `astopy.table.Table` for all responses present in a specific
+        `~sunpy.net.fido_factory.UnifiedResponse` object. They can then be used
+        to perform key-based indexing of QueryResponses
+
+        Returns
+        -------
+        tables : `astropy.table.Table`, list
+            a list or a single table consisting of data from the QueryResponse
+        """
+
+        tables = []
+        for block in self.responses:
+            tables.append(block.build_table())
+        return tables[0] if len(tables) == 1 else tables
```

Comment:
So we either get a list of tables or just one table?

Reply:
I'm not a fan of any API that returns different things at different times. 

@sashank27 Is the returning of list only affected by the user input? In other words, can the user know that they will get a list or table? (Even then this is not elegant, but probably justifiable)

Reply:
From your tests below, it is not very obvious to me when the `.table` will return a `Table` or a list.

I vote for renaming to `.tables` and always returning a list.

Reply:
If there is only one `QueryResponse` then only a `Table` is returned, otherwise, list.
Yeah, it seems that this is not evident from the Users point of view.
Will shift it to always return a list.